Update docs index to link out to Azure

cartermp opened the pull request:
This tab gets a nontrivial amount of clicks per month, but the actual content under the `#cloud` pivot is out of date and has poor engagement. There is an Azure .NET index page in the Azure docs which is actively improved, with the intention of it being our home page for Azure and .NET. I propose that we link to that page here.

Separately, I'd also like to remove each of these cards, since they would effectively be dead HTML with this change.

cc @Andrew-MSFT

cartermp committed:
Update docs index to link out to Azure

This tab gets a nontrivial amount of clicks per month, but the actual content under the `#cloud` pivot is out of date and has poor engagement. There is an Azure .NET index page in the Azure docs which is actively improved, with the intention of it being our home page for Azure and .NET. I propose that we link to that page here.

Separately, I'd also like to remove each of these cards, since they would effectively be dead HTML with this change.

mairaw commented:
@sarah-barrett can you comment on this change for our .NET hub page? This would make one of the tabs go straight to the destination instead of showing the cards.

cartermp commented:
@svick The tab is clicked a nontrivial amount of times, but each individual boxy-link is not clicked much at all (and the Cloud Services link sends people down a path to a legacy product we don't want people using anymore). Given that those sets of links are not maintained (nor were they particularly curated), I think it's far better to link out to the resource that we're actively maintaining and improving.

mairaw commented:
I agree that if we make the tab link go somewhere else, we should remove the card links from underneath.
But I'd like to get @sarah-barret's input here as our AI person given that one tab would behave very differently than the others. 
The other option would be to just keep one card that goes to the other portal and keep the tab functionality.

Andrew-MSFT commented:
pinging this PR

sarah-barrett commented:
Tabs should not take you to a different page. If you want to promote a link, I'd recommend putting the card for it on the default tab, or an approach like that.

cartermp committed:
Update to undo link change and remove cloud services

Cloud services are no longer where we want to point people. Also move the F# one to be last.

mairaw reviewed (approved):
LGTM
